src/connectivity.jl

```diff
@@ -733,3 +733,66 @@ function isgraphical(degs::Vector{<:Integer})
     end
     return true
 end
+
+
+"""
+    isdigraphical(indegree_sequence, outdegree_sequence)
+
+Return true if the degree sequences `indegree_sequence` and `outdegree_sequence` satisfy the (simple) digraph realization problem.
+
+Two sequences of integers of integers satisfy the (simple) digraph realization problem, if there exists a simple directed graph where the indegrees of its vertices form that first sequence and the outdegrees the second.
+
+
+### Implementation Notes
+According to Fulkerson–Chen–Anstee theorem, a sequence ``\\{(a_1, b_1), ...,(a_n, b_n)\\}`` (sorted in descending order of a) is graphic iff the sum of vertex degrees is even and  ``\\sum_{i = 1}^{n} a_i = \\sum_{i = 1}^{n} b_i\\}`` and the sequence obeys the property -
+```math
+\\sum_{i=1}^{r} a_i \\leq \\sum_{i=r+1}^n min(r-1,b_i) + \\sum_{i=r+1}^n min(r,b_i)
+```
+for each integer r <= n-1
+"""
+function isdigraphical(indegree_sequence::Vector{<:Integer}, outdegree_sequence::Vector{<:Integer} )
+
+    n = length(indegree_sequence)
+
+    n == length(outdegree_sequence) || return false 
```

Comment:
Wouldn't it be more appropriate to throw an exception in this case?

Reply:
Hi @simonschoelly, we've just used the same convention adopted for `isgraphical`.

Reply:
Yes, [done](https://github.com/JuliaGraphs/Graphs.jl/pull/186/commits/045b8b55533118795c9e21482ad954a660c9f4a1).